src/Symfony/Component/Validator/Constraints/Choice.php

```diff
@@ -69,7 +69,7 @@ public function __construct(
         if (\is_array($choices) && \is_string(key($choices))) {
             $options = array_merge($choices, $options);
         } elseif (null !== $choices) {
-            $options['choices'] = $choices;
+            $options[$this->getDefaultOption()] = $choices;
```

Comment:
This is reasonable change and we should apply this to other constraints with a default option as well. Do you want to check this or should I?

Reply:
i will try to do this today